tests/integration/test_metrics.py

```diff
@@ -813,47 +797,19 @@ def test_transaction_metrics_extraction_external_relays(
         "public_key": mini_sentry.get_dsn_public_key(project_id),
         "transaction": "root_transaction",
     }
-    external.send_transaction(project_id, tx, item_headers, trace_info)
+    external.send_transaction(project_id, tx, None, trace_info)
 
     envelope = mini_sentry.captured_events.get(timeout=3)
     assert len(envelope.items) == 1
     tx_item = envelope.items[0]
 
-    if expect_extracted_header:
-        assert tx_item.headers["metrics_extracted"] is True
-    else:
-        assert "metrics_extracted" not in tx_item.headers
+    assert "metrics_extracted" not in tx_item.headers
 
     tx_item_body = json.loads(tx_item.get_bytes().decode())
     assert (
         tx_item_body["transaction"] == "/organizations/:orgId/performance/:eventSlug/"
     )
 
-    if expect_metrics_extraction:
-        metrics_envelope = mini_sentry.captured_events.get(timeout=3)
-        assert len(metrics_envelope.items) == 1
-        m_item_body = json.loads(metrics_envelope.items[0].get_bytes().decode())
-        assert len(m_item_body) == 3
-        m_item_body_sorted = sorted(m_item_body, key=lambda x: x["name"])
-        assert (
-            m_item_body_sorted[2]["name"] == "d:transactions/duration_light@millisecond"
-        )
-        assert (
-            m_item_body_sorted[2]["tags"]["transaction"]
-            == "/organizations/:orgId/performance/:eventSlug/"
-        )
-        assert m_item_body_sorted[1]["name"] == "d:transactions/duration@millisecond"
-        assert (
-            m_item_body_sorted[1]["tags"]["transaction"]
-            == "/organizations/:orgId/performance/:eventSlug/"
-        )
-        assert (
-            m_item_body_sorted[0]["name"]
-            == "c:transactions/count_per_root_project@none"
-        )
-        assert m_item_body_sorted[0]["tags"]["transaction"] == "root_transaction"
-        assert m_item_body_sorted[0]["value"] == 1.0
-
```

Comment:
Can we restore this test case and all its variants by adding a "sample-all" dynamic sampling rule at the beginning?